test/Microsoft.Azure.SignalR.Management.Tests/IInternalServiceHubContextFacts.cs

```diff
@@ -54,6 +56,34 @@ public async Task CreateServiceHubContext_WithReferenceNotEqualEndpoints()
             Assert.True(innerEndpoints.SequenceEqual(hubEndpoints.Take(selectedCount), ReferenceEqualityComparer.Instance));
         }
 
+        [InlineData(ServiceTransportType.Persistent)]
```

Comment:
plan to add transient mode or remove it?

Reply:
Plan to add transient mode.